packages/remark-math/types/index.d.ts

```diff
@@ -1,9 +1,13 @@
 import {Plugin} from 'unified' // eslint-disable-line import/no-extraneous-dependencies
 
-interface RemarkMathOptions {
-  inlineMathDouble?: boolean
+declare namespace remarkMath {
+  type Math = Plugin<[RemarkMathOptions?]>
+
+  interface RemarkMathOptions {
+    inlineMathDouble?: boolean
+  }
```

Comment:
stylistically, it can be nice for types to be defined before they are used (even if they don't need to be).

```suggestion
  interface RemarkMathOptions {
    inlineMathDouble?: boolean
  }

  type Math = Plugin<[RemarkMathOptions?]>
```